### PR TITLE
Add oVirt ovirt_affinity_labels and ovirt_affinity_labels_facts modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

- ovirt_affinity_labels 
- ovirt_affinity_labels_facts

##### ANSIBLE VERSION

```
ansible 2.3.0
```

##### SUMMARY

This modules add support to work with oVirt affinity labels.

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @ryansb

File: cloud/ovirt/ovirt_affinity_labels.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_affinity_labels
+short_description: Module to affinity labels in oVirt
+version_added: "2.3"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "This module manage affinity labels in oVirt. It can also manage asignments
```
Comment (mwperina) [inline cloud/ovirt/ovirt_affinity_labels.py]: asignments -> assignments

File: cloud/ovirt/ovirt_affinity_labels.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+       of those lables to hosts and VMs."
```
Comment (mwperina) [inline cloud/ovirt/ovirt_affinity_labels.py]: lables -> labels

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
Looks good to me! :+1:

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
shipit

**Review by mwperina — APPROVED** [review]
:+1:

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request merged**